### PR TITLE
Fixed multipage

### PR DESCRIPTION
**Pull request by chack1172**
Fixed multipage after previous PR.

### CODE REVIEW

File: inc/plugins/simplelikes.php [hunk 1]
```diff
@@ -646,10 +646,9 @@ function simplelikesMisc()
 				$where_sql .= " AND p.fid NOT IN ({$inactiveforums})";
 			}
 
-			$count = (int)$db->fetch_field(
-				$db->simple_select('post_likes', 'COUNT(*) AS count', "user_id = {$userId}"),
-				'count'
-			);
+			$queryString = "SELECT COUNT(*) AS count FROM %spost_likes l LEFT JOIN %sposts p ON (l.post_id = p.pid) WHERE l.user_id = {$userId}{$where_sql}";
```
Comment (euantorano) [inline inc/plugins/simplelikes.php]: The `JOIN`s for these queries should probably be `INNER JOIN` as we only want to show likes where both likes and posts exist.

Comment (chack1172) [reply inc/plugins/simplelikes.php]: You're right. But I saw that all your queries are using `LEFT JOIN` so we should change all of them, right?

Comment (euantorano) [reply inc/plugins/simplelikes.php]: Yep, we should. I just remembered when I saw this PR 😛

Comment (chack1172) [reply inc/plugins/simplelikes.php]: Ok and what about soft deleted posts and threads? Should we count the likes?

Comment (chack1172) [reply inc/plugins/simplelikes.php]: Anyway I think that you should merge this pull request at the moment. We change queries in another pull.

Comment (euantorano) [reply inc/plugins/simplelikes.php]: Yeah, we should probably ignore deleted/soft deleted threads/posts. I'll merge this now and can fix it later.

**Pull request merged**